### PR TITLE
Product page

### PR DESCRIPTION
**Pull request by swstica**

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -159,7 +160,7 @@ section{
 
 footer{
```
Comment (prankurpandeyy) [inline index.css]: please avoid using direct global tags

**Review by prankurpandeyy — CHANGES_REQUESTED** [review]

**Comment by prankurpandeyy** [comment]
your logo is aligning with the list items in header give some margin to them

**Review by prankurpandeyy — CHANGES_REQUESTED** [review]
rest is looking good , suggested some changes

**Review by varun1233 — APPROVED** [review]
Code Looks code, just work on the changes suggested by Prankur, after that it's good to merge

File: product-page/product.css [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    width: 34rem;
+}
+.product-actions{
+    gap: 1rem;
+}
+.dropdown{
+    gap: 1rem;
+}
+.cart-btn{
+    color: var(--white-color);
```
Comment (Rishabh-078) [inline product-page/product.css:32]: It was suggested not to use the actual name of the color as a variable name.

**Review by Rishabh-078 — APPROVED** [review]
The code is looking fine, just had a suggestion that was pointed out.
You can go ahead and merge it.

**Review by prankurpandeyy — APPROVED** [review]

**Pull request merged**